tests/utils/mls_utils/mod.rs

```diff
@@ -0,0 +1,297 @@
+#![allow(dead_code)]
```

Comment:
The way I understand https://github.com/rust-lang/rust/issues/46379, non-test modules in `tests` get compiled individually with each test. At least on my machine, this means I get warnings, because not every test uses every function of the framework and reports the non-used functions as dead code. Not sure if there is a really good solution to this.